pkg/controllers/dynakube/logmonitoring/logmonsettings/reconciler.go

```diff
@@ -79,7 +79,16 @@ func (r *reconciler) checkLogMonitoringSettings(ctx context.Context) error {
 	if err != nil {
 		setLogMonitoringSettingError(r.dk.Conditions(), conditionType, err.Error())
 
-		return errors.WithMessage(err, "error when creating log monitoring setting")
+		if !r.dk.ActiveGate().IsKubernetesMonitoringEnabled() && r.dk.Status.KubernetesClusterMEID == "" {
```

Comment:
IMO you should just simply check [in the beginning](https://github.com/Dynatrace/dynatrace-operator/blob/0b6cc5e10b1140eedc7385cfca607032e06ea8ad/pkg/controllers/dynakube/logmonitoring/logmonsettings/reconciler.go#L40) that  `r.dk.Status.KubernetesClusterMEID != ""` otherwise do nothing.

Your request will never go through if `r.dk.Status.KubernetesClusterMEID == ""`

Reply:
you can/should still log something in that case

Reply:
I agree this is nicer. 

Do you mean by 

> you can/should still log something in that case

to add a log message [here](https://github.com/Dynatrace/dynatrace-operator/blob/0b6cc5e10b1140eedc7385cfca607032e06ea8ad/pkg/controllers/dynakube/logmonitoring/logmonsettings/reconciler.go#L42)? Or am I wrong?

Reply:
proposed solution: https://github.com/Dynatrace/dynatrace-operator/pull/4174/commits/f4c3bb120f1bcd3de07004fae437f1f56236ea12